doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -407,8 +345,6 @@ If present, the error payload is a JSON object including the following fields:
 
 + `error` (required, string): a textual description of the error.
 + `description` (optional, string): additional information about the error.
-+ `affectedItems` (optional, array[string]): a list of elements affected by the error.
-  Depending on the operation, it may refer to entities, registrations or subscriptions.
```

Comment:
`affectedItems` has been actually used in Orion reference implementation. In addition, some partners has raised concerns about this field.

Thus, being conservative, it is better not saying anything about this in this release candidate (next ones will do).